src/Services/Spellcheckers/InMemorySpellchecker.php

```diff
@@ -40,7 +40,7 @@ public static function default(): self
      */
     public function check(string $text): array
     {
-        $misspellings = $this->filterWhitelistedWords(iterator_to_array($this->aspell->check($text)));
+        $misspellings = $this->filterWhitelistedWords(iterator_to_array($this->aspell->check($text, $this->config->languages)));
```

Comment:
I stumbled across the fact that Aspell only supports one language: 
https://github.com/tigitz/php-spellchecker/blob/dbbf94efc9761505a9fa9583537b5f9d1b53260d/src/Spellchecker/Aspell.php#L28

So if we let the user configure an array, it can cause an unexpected exception here!

Reply:
Yeah - it says one language, I have seen that. The config is currently an array as that is what is expected by Aspell.

Agreed - we could make our config take only a string and wrap that in an array, but I went with this for now.

Reply:
Yes, this was what I had in mind as well. Maybe that's something you could take a look at?